### ISSUE TITLE
Prepare syllabuses for low data access

### ISSUE THREAD

**Issue by sheenarbw** [Umuzi-org/tech-department#426]
COVID-19 is coming and there is a very real chance that Umuzi will need to close it's doors for a litttle while in order to keep this community safe. 

We want to make sure that our recruits can keep learning and growing while they are stuck at home. This is going to be hella expensive. 

One way we want to reduce costs is by reducing the amount of internet infrastructure people need. If they need less data access per person then things get cheaper.  Also, any work we do here will be useful inside Umuzi - we've had our fair share of internet problems.

Here's what we need:

Go to each department head and get a list of all their content, including all the necessary links and access to that content. Don't worry about approaching the tech department, for the most part that is already very downloadable. 

Figure out for each department:
- what can't be made offline-friendly (eg: the data science dept uses DataCamp, it simply wont work offline)
- what can be made offline friendly 

Thoughts:
Some of that content will be easy to download. Eg: there might be some googlle docs and pdfs and that sort of thing that are easy to just download.

Some content will need a little bit more work. Eg: Youtube videos (we can write a python script that does bulk downloads). Online articles (eg Medium) aren't as tidy for downloads)

Basically we need to use the output of this task to come up with a plan of action so that we can get just copy syllabus material to people's PCs.
